agent/structs/config_entry_gateways.go

```diff
@@ -90,6 +90,9 @@ type IngressService struct {
 	RequestHeaders  *HTTPHeaderModifiers `json:",omitempty" alias:"request_headers"`
 	ResponseHeaders *HTTPHeaderModifiers `json:",omitempty" alias:"response_headers"`
 
+	// MaxConnections is the max connections that is allowed to a service
```

Comment:
```suggestion
	// MaxConnections is the max number of connections that is allowed to a service
```

Can we also clarify what the zero value here would mean?

Reply:
We may want to keep this `*int32`; I never remember how `json:",omitempty"` works with zero values of primitives